tembo-stacks/src/stacks/specs/analytics.yaml

```diff
@@ -23,21 +22,45 @@ postgres_config:
   - name: pg_stat_statements.track
     value: top
   - name: shared_preload_libraries
-    value: pg_stat_statements,pg_cron
+    value: pg_stat_statements, pg_cron, pg_later, pg_parquet
   - name: track_io_timing
     value: 'on'
   - name: wal_level
     value: logical
+  - name: pglater.host
+    value: postgresql:///postgres?host=/controller/run
 trunk_installs:
+  - name: hydra_columnar
+    version: 1.1.1
   - name: pg_stat_statements
     version: 1.10.0
   - name: pg_partman
     version: 5.0.1
   - name: pg_cron
     version: 1.6.2
+  - name: pg_later
+    version: 0.1.1
```

Comment:
latest is v0.3.0. We will need to upgrade this version in order to add pg17 support.

Reply:
Will update.